esileapclient/tests/unit/common/test_utils.py

```diff
@@ -0,0 +1,73 @@
+import unittest
+from esileapclient.common import utils
+
+
+class TestUtils(unittest.TestCase):
+
+    def setUp(self):
+        self.nodes = [
+            {'properties': {'cpus': '40', 'memory_mb': '131072'}},
+            {'properties': {'cpus': '80', 'memory_mb': '262144'}},
+            {'properties': {'cpus': '20', 'memory_mb': '65536'}},
+        ]
+        self.parser = utils.PropertyFilterParser()
+
+    def test_convert_value(self):
+        self.assertEqual(self.parser.convert_value('10'), 10)
+        self.assertEqual(self.parser.convert_value('10.5'), 10.5)
+
+    def test_parse_property_filter(self):
+        key, op, value = self.parser.parse_property_filter('cpus>=40')
+        self.assertEqual(key, 'cpus')
+        self.assertEqual(op, utils.PropertyFilterParser.OPS['>='])
+        self.assertEqual(value, 40)
+
+        key, op, value = self.parser.parse_property_filter('memory_mb<=131072')
+        self.assertEqual(key, 'memory_mb')
+        self.assertEqual(op, utils.PropertyFilterParser.OPS['<='])
+        self.assertEqual(value, 131072)
+
+        with self.assertRaises(ValueError):
+            self.parser.parse_property_filter('invalid_filter')
+
+    def test_node_matches_property_filters(self):
+        filters = [
+            self.parser.parse_property_filter('cpus>=40'),
+            self.parser.parse_property_filter('memory_mb>=131072')
+        ]
+        self.assertTrue(utils.node_matches_property_filters(
+            self.nodes[1], filters))
+        self.assertFalse(utils.node_matches_property_filters(
+            self.nodes[2], filters))
```

Comment:
You also want to test the case where the property doesn't exist (where `key not in node['properties']`)